test/Shared/MsBuildProjectStrings.cs

```diff
@@ -184,14 +184,19 @@ public void ConfigureServices(IServiceCollection services)
         public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
         {
             app.UseStaticFiles();
-
+            CS7_method(out var i);
```

Comment:
Just to make sure that in the test project scaffolding would work if there are any C# 7 specific features used. 
The snippet above is used as a template to add a startup class to the test project.